src/main/java/com/github/princesslana/eriscasper/rest/guild/CreateGuildBanRequest.java

```diff
@@ -0,0 +1,23 @@
+package com.github.princesslana.eriscasper.rest.guild;
+
+import com.github.princesslana.eriscasper.util.QueryStringBuilder;
+import org.immutables.value.Value;
+
+/**
+ * @see <a
+ *     href="https://discordapp.com/developers/docs/resources/guild#create-guild-ban-query-string-params">
+ *     https://discordapp.com/developers/docs/resources/guild#create-guild-ban-query-string-params</a>
+ */
+@Value.Immutable
+public interface CreateGuildBanRequest {
+  Integer getDeleteMessageDays();
+
+  String getReason();
+
+  default String toQueryString() {
+    return new QueryStringBuilder()
+        .add("reason", getReason())
+        .add("delete_message_days", getDeleteMessageDays().toString())
```

Comment:
Shouldn't this make use of addLong?

Reply:
it's non optional

Reply:
You could call with Optional.of(getDeleteMessageDays()) or add an `addLong(String, Long)` method to QueryStringBuilder.

Reply:
Alright, I'll do that.